client/tests/integration_test.rs

```diff
@@ -330,6 +348,130 @@ mod get_state_root_hash {
         let server_handle = MockServerHandle::spawn_without_params(GetStateRootHash::METHOD);
         assert_eq!(server_handle.get_state_root_hash(""), Ok(()));
     }
+
+    #[tokio::test(threaded_scheduler)]
+    async fn should_succeed_with_block_id_of_height() {
```

Comment:
```suggestion
    async fn should_succeed_with_valid_block_height() {
```

Please also rename `should_succeed_with_empty_block_id()` to `should_succeed_with_empty_block_hash()`.

Also, could you change the naming of the tests to be consistent.  I'd prefer if we didn't repeat the module name as the test prefix, so for example, `get_block::get_block_should_succeed_with_valid_block_id()` should be renamed to `get_block::should_succeed_with_valid_block_id()`